lib/version.js

```diff
@@ -112,6 +113,8 @@ class NodeVersion {
 				return 'x86';
 			case '64':
 			case 'x64':
+			case 'ppc64':
+			 return (os.endianness() == 'LE' ? 'ppc64le' : 'ppc64')
```

Comment:
This case needs to be moved below the `return` in the `amd64' case, because currently the `x64` case is falling through to this one. I didn't notice this until I saw that it's breaking a bunch of unit tests.

Reply:
updated PTAL